packages/subapp-web/lib/start.js

```diff
@@ -85,6 +93,8 @@ module.exports = function setup() {
             if (group !== "_") {
               mapCtx.assertNoFailure();
 
+              startTimes[`load-grp-${group}`] = Date.now();
```

Comment:
if `startTimes` is only used within this closure, then it can simply be a variable.